source/game/render/KartModel.cc

```diff
@@ -38,7 +39,7 @@ void KartModel::vf_1c() {
 
     _54 += fVar2 * (xStick - _54);
 
-    if (_58 < -_54 || _54 < _58) {
+    if (local_f31 < -_54 || _54 < local_f31) {
```

Comment:
Something tells me we can reduce this somehow using `EGG::Mathf::abs()` but I can't quite put my finger on it

Reply:
i did abs(local_f31) > abs(_54), but it seems far from ghidra so i put a label comment so ppl know what it is

Reply:
reverted